src/essentials/index.ts

```diff
@@ -1,3 +1,4 @@
 export { Elevation } from './Elevation/Elevation';
 export { Spaces } from './Spaces/Spaces';
 export { Breakpoints, MediaQueries } from './Breakpoints/Breakpoints';
+export type { ReadCssColorVariable } from './Colors/types';
```

Comment:
Let's also expose all wave css variables
```suggestion
export type { ReadCssColorVariable, ReadCssVariable } from './Colors/types';
```

Reply:
I've ended up creating a `BareColorsSchema` following the `SemanticColorsSchema` and appropriate types based on that, please take a look again when you have time 🙏🏽

Reply:
I intentionally do not expose bare values because they should only be used as an escape hatch

Reply:
I understand and agree with the intention, but since projects are currently using `Colors` as a type I need a replacement type for that, I only see as options either exposing a type representing the bare colors as css variables or a generic `string`.

Reply:
I've removed the bare colors schema to not expose bare colors